test/test-app/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
app_component_test.ts

I don't have any objection for writing test, but I don't want to you focus too much on it. Since this is already an app for test, if you write test, you should violate the compiler rules in test.

Reply:
I agree, I'll delete all the other test files for now and just leave app_component_test.ts as the only test file, which I put some anti-patterns in as well.